lib/auth/apiserver.go

```diff
@@ -2278,7 +2278,7 @@ func (s *APIServer) getClusterAuthPreference(auth ClientI, w http.ResponseWriter
 }
 
 type setClusterAuthPreferenceReq struct {
-	ClusterAuthPreference json.RawMessage `json:"cluster_auth_prerference"`
+	ClusterAuthPreference json.RawMessage `json:"cluster_auth_preference"`
```

Comment:
The typo is unfortunate but I think changing this tag might result in broken compatibility b/w teleport clients and servers of different versions.

Reply:
Occurred to me it could be an issue, thanks for pointing it out!